pkg/controller/configuration/controller.go

```diff
@@ -118,6 +150,10 @@ func (r *Reconciler) reconcileConfiguration(ctx context.Context, config *configa
 	_ = target.GetAspect(&mastership)
 	targetMastershipTerm := configapi.MastershipTerm(mastership.Term)
 
+	if config.Status.State == configapi.ConfigurationState_CONFIGURATION_COMPLETE {
```

Comment:
Should this be `|| config.Status.State == configapi.ConfigurationState_CONFIGURATION_FAILED`?